[IMP] live_chat: overhaul documentation update (#601)
odoo/documentation-user: livechat/overview/responses.rst

=============================
Commands and Canned Responses
=============================

Using canned responses can help you save time and have a previous, well-thought response, to some
of your most common questions and comments.

Use commands
============

Commands are shortcuts for creating certain actions within the chat window:
#. **/help**: shows a help message.
#. **/helpdesk**: creates a helpdesk ticket.
#. **/helpdesk_search**: searches for a helpdesk ticket.
#. **/history**: shows the last 15 visited pages.
#. **/lead**: creates a new lead.
#. **/leave**: leaves the channel.

.. important::
   | -For *helpdesk tickets*: make sure the application is installed on your database, and the
     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`,

>      is enabled.

Reviewer: ```suggestion
   | - For *helpdesk tickets*: make sure the application is installed on your database and the
     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
     is enabled.
```

=============================
Commands and Canned Responses
=============================

Using canned responses can help you save time and have a previous, well-thought response, to some
of your most common questions and comments.

Use commands
============

> Commands are shortcuts for creating certain actions within the chat window:

Reviewer 2: ```suggestion
Commands are shortcuts that do specific actions within the chat window:
```
